openapi/index.yaml

```diff
@@ -756,7 +756,7 @@ definitions:
           created_at:
             $ref: '#/definitions/Timestamp'
           content:
-            $ref: '#/definitions/MessageContent'
+            $ref: '#/definitions/NewMessageContent'
```

Comment:
I see this change to come from template rendering, I just wonder why it happened.

Reply:
This change comes from [here](https://github.com/pagopa/io-functions-commons/pull/218/files#diff-390ee41a01068fe2e94192fa8caf0b01f8ad5e5a71545ae89874cf453348b334R267) released with `v21.7.3` of `io-functions-commons`